marlowe-runtime/cip30-demo/Main.hs

```diff
@@ -1,14 +1,30 @@
 {-# LANGUAGE QuasiQuotes #-}
+{-# LANGUAGE RankNTypes #-}
```

Comment:
I don't see anything in here that needs RanNTypes - is this because of TH code that Yesod is generating?

Reply:
It's because of `RunClient` in
```haskell
runTxJobClient :: Socket.HostName -> Socket.PortNumber -> RunClient IO (JobClient Transaction.Api.MarloweTxCommand)
```
If I go with the same structure you did where it's wrapped in a `newtype` then it also needs the `RankNTypes` extension to work.